owncloudData/src/test/java/com/owncloud/android/data/webfinger/datasource/implementation/OCRemoteWebFingerDatasourceTest.kt

```diff
@@ -0,0 +1,134 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Aitor Ballesteros Pavón
+ *
+ * Copyright (C) 2023 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.data.webfinger.datasource.implementation
+
+import com.owncloud.android.data.ClientManager
+import com.owncloud.android.data.webfinger.datasources.implementation.OCRemoteWebFingerDatasource
+import com.owncloud.android.domain.webfinger.model.WebFingerRel
+import com.owncloud.android.lib.common.OwnCloudClient
+import com.owncloud.android.lib.common.operations.RemoteOperationResult
+import com.owncloud.android.lib.resources.webfinger.services.implementation.OCWebFingerService
+import com.owncloud.android.testutil.OC_ACCESS_TOKEN
+import com.owncloud.android.testutil.OC_ACCOUNT_ID
+import com.owncloud.android.testutil.OC_SECURE_SERVER_INFO_BASIC_AUTH
+import com.owncloud.android.utils.createRemoteOperationResultMock
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+class OCRemoteWebFingerDatasourceTest {
+
+    private lateinit var ocRemoteWebFingerDatasource: OCRemoteWebFingerDatasource
+
+    private val clientManager: ClientManager = mockk(relaxed = true)
+    private val ownCloudClient: OwnCloudClient = mockk(relaxed = true)
+    private val ocWebFingerService: OCWebFingerService = mockk()
+    private val listString: List<String> = listOf(
```

Comment:
I would give it a more meaningful name like `urls` or so

Reply:
Done